gatsby-node.js

```diff
@@ -43,4 +51,56 @@ exports.createPages = async ({ actions }) => {
       },
     });
   });
+
+  return graphql(`
+    {
+      allPage {
+        edges {
+          node {
```

Comment:
The Program page needs to know the data that's defined [here](https://github.com/edx/frontend-app-learner-portal/pull/77/files#diff-fda05457e393bada716f508859bfc604R46), and ideally with the same names. We should also delete [this](https://github.com/edx/frontend-app-learner-portal/pull/77/files#diff-fda05457e393bada716f508859bfc604R21) and [this](https://github.com/edx/frontend-app-learner-portal/pull/77/files#diff-fda05457e393bada716f508859bfc604R36) so it's not creating pages for that hardcoded data.